release/bazelrc.tpl

```diff
@@ -0,0 +1,124 @@
+# This .bazelrc file contains all of the flags required for the toolchain,
+# Remote Build Execution, and the Bazel Build Results UI. Specific flags in
+# your Bazel command allow you to use only the remote build, to use only the
+# results UI, or to use them both together.
+#
+# This .bazelrc file also contains all of the flags required for the local
+# docker sandboxing.
```

Comment:
Add note that this is a generated file and should not be modified directly (and point to source release/bazelrc.tpl as the template)

Reply:
Done.